libc/CMakeLists.txt

```diff
@@ -702,7 +727,7 @@ if (CMAKE_C_COMPILER_ID MATCHES GNU)
     -Wno-unused-function
     -Wno-unused-value
     -Wno-unused-variable)
-elseif (CMAKE_C_COMPILER_ID MATCHES Clang)
+elseif (CMAKE_C_COMPILER_ID MATCHES Clang OR USE_CLANGW)
```

Comment:
Ah and this is where having an option with `CLANG` in the name makes more sense. Maybe... `BUILD_ELF_ON_WINDOWS_WITH_CLANG`. It's long, but accurate! 😂

Reply:
I am leaning towards going in with one suggestion (what ever that is) and then renaming it later as we play with it more.